clusterloader2/pkg/measurement/common/network-policy/manifests/dep-test-client-pod-creation.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.Name}}
+  namespace: {{.Namespace}}
+  labels:
+    test: {{.TestClientLabel}}
+    type: {{.TypeLabelValue}}
+spec:
+  replicas: 1
```

Comment:
The way it's implemented now, it wouldn't make sense to have multiple replicas of the same pod, because it will target the same pods.
I want to deploy exactly one pod per target namespace, and I use a deployment with 1 replica to do so.

The pods are intended to run on a separate set of nodes that is specific for hosting test client pods, to avoid interfering with the load test.

There is a node selector specified, and a comment for it:
```
      # Use separate nodes to avoid consuming CPU/Memory resources on default
      # nodes where all deployments of the performance test run.
```